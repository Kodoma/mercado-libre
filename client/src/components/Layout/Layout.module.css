.container {
    min-height: 100vh;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 100vh;
}
.main {
    background-color: #eeeeee;
    min-height: 1000px;
    height: 100%;
    padding: 10px 0;
    flex: 1;
    display: flex;
    justify-content: center;
    flexdirection: column;
    justifycontent: center;
    alignitems: center;
}
