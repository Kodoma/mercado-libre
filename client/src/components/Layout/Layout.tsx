import styles from './Layout.module.css';
import React from 'react';
import HeadSeo from '../Head/Head';
import Header from '../Header/Header';
import CssBaseline from '@material-ui/core/CssBaseline';
import { unstable_createMuiStrictModeTheme } from '@material-ui/core/styles';
import Breadcrumb from '../BreadCrumb/Breadcrumb';

const Layout: React.FunctionComponent<LayoutProps> = ({ children }) => {
    return (
        <React.Fragment>
            <HeadSeo />
            <CssBaseline />
            <Header />
            <main className={styles.main}>{children}</main>
        </React.Fragment>
    );
};

export default Layout;

interface LayoutProps {
    children: any;
}
