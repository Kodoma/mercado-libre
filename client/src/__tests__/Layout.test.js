import React from 'react';
import { shallow } from 'enzyme';
import Layout from "../components/Layout/Layout";

describe('tests Layout', () => {
    it('should render my Layout component', () => {
        const component = shallow(<Layout/>);
        expect(component.getElements()).toMatchSnapshot();
    });
});