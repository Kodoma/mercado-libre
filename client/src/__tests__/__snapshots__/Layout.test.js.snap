// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__tests__ Layout should render my Layout component 1`] = `
Array [
  <React.Fragment>
    <Memo(HeadSeo) />
    <WithStyles(CssBaseline) />
    <Memo(Header) />
    <main
      className="main"
    />
  </React.Fragment>,
]
`;
